examples/tts/conf/fastpitch_align.yaml

```diff
@@ -0,0 +1,183 @@
+name: "FastPitch"
+sample_rate: 44100
+train_dataset: ???
+validation_datasets: ???
+prior_folder: ???
+
+model:
+  learn_alignment: true
+  n_speakers: 1
+  symbols_embedding_dim: 384
+  max_token_duration: 75
+  n_mel_channels: 80
+  pitch_embedding_kernel_size: 3
+  n_window_size: 2048
+  n_window_stride: 512
+
+  train_ds:
+    dataset:
+      _target_: "nemo.collections.asr.data.audio_to_text.AudioToCharWithPriorAndPitchDataset"
+      manifest_filepath: ${train_dataset}
+      max_duration: null
+      min_duration: 0.1
+      int_values: false
+      normalize: true
+      sample_rate: ${sample_rate}
+      trim: false
+      sup_data_path: ${prior_folder}
+      n_window_stride: ${model.n_window_stride}
+      n_window_size: ${model.n_window_size}
+      pitch_fmin: 80
+      pitch_fmax: 640
+      pitch_avg: 216.23100173462166
+      pitch_std: 46.24582950436719
+      vocab:
+        notation: phonemes
+        punct: true
+        spaces: true
+        stresses: true
+        add_blank_at: None
+        pad_with_space: True
+        chars: true
+        improved_version_g2p: true
+    dataloader_params:
+      drop_last: false
+      shuffle: true
+      batch_size: 32
+      num_workers: 12
+
+
+  validation_ds:
+    dataset:
+      _target_: "nemo.collections.asr.data.audio_to_text.AudioToCharWithPriorAndPitchDataset"
+      manifest_filepath: ${validation_datasets}
+      max_duration: null
+      min_duration: null
+      int_values: false
+      normalize: true
+      sample_rate: ${sample_rate}
+      trim: false
+      sup_data_path: ${prior_folder}
+      n_window_stride: ${model.n_window_stride}
+      n_window_size: ${model.n_window_size}
+      pitch_fmin: 80
+      pitch_fmax: 640
+      pitch_avg: 216.23100173462166
+      pitch_std: 46.24582950436719
```

Comment:
I would suggest to convert all values related to audio params according to LJ Speech.

Reply:
done